### DISCUSSION TITLE
set default style for books

### DISCUSSION THREAD

**Discussion by TomW1605** [koreader/koreader#11129] [Q&A] [answered]
is there a way to set the default style (in my case left justified and no hyphens) for new books? i have tried saving the document settings as defualt within a book i have the right style set on but it dosnt seem to work (either with new books or reseting existing books to default).

**Comment by poire-z** [comment]
If you are talking about "Style tweaks", long-press on the one you are intested, to either 1) make them default to apply on all books, or 2) "Show in action list" to allow associating them to gestures (and maybe profiles, and maybe so that they are taken if you use "Document settings> Save document settings as default", not sure).

**Reply by TomW1605** [reply]
thanks, that worked great for the left justification that is a style tweek. but the hyphenation is under Typography Rules and dosnt seem to have a long press action

**Reply by poire-z** [reply, marked as answer]
> but the hyphenation is under Typography Rules and dosnt seem to have a long press action

It is a submenu, so maybe something inside it is explicite enough and does support long-press :)

![image](https://github.com/koreader/koreader/assets/24273478/bee92d13-0efb-44ae-addd-11f29dc454ba)

**Reply by TomW1605** [reply]
thanks, not sure why that i didnt find that before (im sure i tried long press on that)
